src/NextjsDistribution.ts

```diff
@@ -400,6 +400,7 @@ export class NextjsDistribution extends Construct {
         // what goes here? static or lambda?
         cachePolicy: lambdaCachePolicy,
         originRequestPolicy: fallbackOriginRequestPolicy,
+        responseHeadersPolicy: staticResponseHeadersPolicy,
```

Comment:
Not sure if this is the right thing to do.

Reply:
i just don't know how safe it is to cache lambda responses by default. that might be a surprising behavior for some application. also think about policies for caching based on path params, cookies, etc. better to leave that to the user to specify IMO

Reply:
without this, the fallback won't have the header response at all :| not sure what the right approach is here.

Reply:
but might be right for dynamic paths though, needs more testing.

Reply:
I believe if the user wants to cache SSR responses they should explicitly configure it with a cache policy because you need to specify the various behaviors and it might lead to surprising results, like caching authenticated pages for example

Reply:
Hmm ok, so it seems like we should give the static asset bucket the cache-control config, and remove it here.